popup.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>KeepTabs Pop-Up</title>

```

Comment:
Popup is a technical term in the domain of Chrome extensions. Give it a title meaningful (and that doesn't sound bad -- popup sounds like an ad) to the user, e.g. KeepTabs Options.